config/application.rb

```diff
@@ -9,6 +9,7 @@ module HerosTeamApp
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 5.1
+    config.autoload_paths << "#{Rails.root}/lib"
```

Comment:
there's no need for this, just remove it

Reply:
for me ideal is to place this file in app/lib folder (not in just lib) folder. but it's up to you

Reply:
file is located within `app/lib`, so there's no need in this line at all for now